src/main/java/seedu/todo/commons/util/TimeUtil.java

```diff
@@ -248,6 +249,16 @@ public boolean isOngoing(LocalDateTime startTime, LocalDateTime endTime) {
         
         return LocalDateTime.now(clock).isAfter(startTime) && LocalDateTime.now(clock).isBefore(endTime);
     }
+    
+    public boolean isToday(ImmutableTask task){
+        if (!task.getStartTime().isPresent() && !task.getEndTime().isPresent()){
+            return false;
+        }
+        LocalDateTime timeToCompareTo = task.getStartTime().isPresent() ?
```

Comment:
Ok so ignore the above comment because clearly you can just do
```
if (startTime.isPresent()) {
    ...
} else if (endTime.isPresent()) {
    ...
} else {
   ...
}